packages/react-tabs/src/tab.constants.ts

```diff
@@ -1,13 +1,73 @@
-// TODO: These constants should be replaced with design tokens
-export const tabPendingDesignTokens = {
-  tabPadding: {
-    medium: '10px',
-    small: '6px',
+// TODO: Constants prefixed with pending should be replaced with design tokens
+// These values were taken from the design token superset figma
+
+import { tokens } from '@fluentui/react-theme';
+
+export const pendingContentSizeTokens = {
+  // Only body sizes are defined here currently
+  body1: {
+    fontSize: tokens.fontSizeBase300,
+    fontWeight: tokens.fontWeightRegular,
+    lineHeight: tokens.lineHeightBase300,
   },
-  indicatorThickness: '2px',
-  gap: { medium: '6px', small: '2px' },
-  contentPadding: {
-    medium: '2px',
-    small: '2px',
+  body1Strong: {
+    fontSize: tokens.fontSizeBase300,
+    fontWeight: tokens.fontWeightSemibold,
+    lineHeight: tokens.lineHeightBase300,
   },
 };
+
+export const pendingSpacingTokens = {
```

Comment:
@miroslavstastny should we have an issue tracking these pending tokens so that we can track what needs to be cleaned up once they exist ?

Reply:
Per tech sync - yes 👍🏻  I'll create one for the tabs control to clean these up.